src/glue/v2/communication.cpp

```diff
@@ -204,9 +204,45 @@ Value ToBoltValue(msgs::Value value) {
     case msgs::Value::Type::Vertex:
     case msgs::Value::Type::Edge:
     case msgs::Value::Type::Path: {
-      throw utils::BasicException("Path, Vertex and Edge not supported!");
+      throw utils::BasicException("Path not supported!");
```

Comment:
`msgs::Value::Type::Edge` and `msgs::Value::Type::Vertex:` will fallthrough here, but we only throw that "Path not supported!"